pkg/skaffold/color/formatter.go

```diff
@@ -19,10 +19,9 @@ package color
 import (
 	"fmt"
 	"io"
-	"os"
 	"strings"
 
-	"golang.org/x/crypto/ssh/terminal"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
```

Comment:
Would it be worth rewritting this code to use `heroku/color` at some point?

Reply:
It's somewhere on my TODO...